WORKSPACE

```diff
@@ -1,35 +1,19 @@
 workspace(name = "jsonnet")
 
-# This local_repository looks silly but it makes io_bazel_rules_jsonnet use
-# _this_ jsonnet, not another downloaded copy.
-local_repository(
```

Comment:
Yeah I dunno what's going on there either.  @Globegitter what do you think

Reply:
Sorry I just saw this now, still struggling with the github "mentioned" ui. Will take a look today.

Reply:
@Globegitter did you get a chance to look into it

Reply:
Sorry, just looked at it now properly. And yeah that comment must be outdated. Bazel today only loads dependencies specified in this workspace file and also it would not allow to load another `jsonnet` workspace anymore anyway.